.azure-pipelines/unix-build.yml

```diff
@@ -7,10 +7,6 @@ steps:
       if [[ $(enable_xtl_complex) == 1 ]]; then
         conda install xtl -c conda-forge
       fi
-      if [[ $(avx512) == 1 ]]; then
-        conda install gcc_linux-64 gxx_linux-64;
-        sh $(Build.SourcesDirectory)/install_sde.sh;
-      fi
```

Comment:
No need to remove everything, only the install of `gcc_linux-64` and `gxx_linux-64`.